modules/doc/content/training/more_detail/MOOSE_2024_10_15-17_UIUC.md

```diff
@@ -6,7 +6,26 @@
 
 !include training/more_detail/generic_moose_training.md
 
-#### Location Details
+#### Getting to Urbana-Champaign
+
+!style! halign=left
++Flying:+ There are three commercial options:
+!style-end!
+
+- Champaign-Urbana (Willard Airport), located 15 minutes from campus. Direct flights to:
+    - Chicago
+    - Dallas
+- Central Illinois Regional Airport (Bloomington), located 1 hour from campus. Direct flights to:
+    - Chicago
+    - Dallas
+    - Atlanta
+    - Detroit
+    - Orlando
+- Ohare International Airport (Chicago), located 2.5 hours from campus
+
+Rental cars are avaialble at the Bloomington and Chicago airports.
```

Comment:
```suggestion
Rental cars are available at the Bloomington and Chicago airports. 
```

Are rental cars currently not available at Willard? I see options for that on their website and Google Maps.

Reply:
They are, I just don't think it's necessary (you can Uber to campus, it's fairly close). But I'll add it in case someone wants it!